package.json

```
@@ -18,9 +17,16 @@
     "zerion"
   ],
   "license": "MIT",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
   "scripts": {
-    "build": "tsc",
-    "start": "node dist/index.js",
+    "build": "rm -fr dist/* && yarn build:esm && yarn build:cjs",
+    "build:esm": "tsc -p tsconfig.esm.json",
+    "build:cjs": "tsc -p tsconfig.cjs.json",    "start": "node dist/index.js",
```

I don't think start means anything anymore here.
```suggestion
    "build:cjs": "tsc -p tsconfig.cjs.json",    
```